crates/optimism/node/src/args.rs

```diff
@@ -13,6 +13,14 @@ pub struct RollupArgs {
     #[arg(long = "rollup.sequencer-http", value_name = "HTTP_URL")]
     pub sequencer_http: Option<String>,
 
+    /// WS endpoint for the sequencer mempool
+    #[arg(long = "rollup.sequencer-ws", value_name = "WS_URL")]
+    pub sequencer_ws: Option<String>,
+
+    /// Endpoint for the sequencer mempool (can be both HTTP and WS)
+    #[arg(long = "rollup.sequencer")]
+    pub sequencer: Option<String>,
```

Comment:
this is now basically the same setting, so we can do two things here

a) unify to just `sequencer-url` and add visible aliases
b) add proper clap conflicts_with attributes

imo a) would be cleaner but since the standard is sequencer-http I'd rather keep that, but we could unify `sequencer-ws` and sequencer into a single instance

Reply:
Decided to go with a)